java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -2104,7 +2104,7 @@ public static void setupComboBox(@Nonnull JmriBeanComboBox inComboBox, boolean i
     /**
      * set the maximum number of rows based on screen size
      *
-     * @param inComboBox
+     * @param inComboBox the combo box
```

Comment:
It is probably better to make this change in another PR.  It doesn't have to do with the new package

Reply:
Perhaps… but a whole PR just for a (minor) JavaDoc tweak?

Reply:
That is certainly appropriate.

Reply:
Regressed (moved to LayoutEditor-development branch)